### DISCUSSION TITLE
How to properly indicate the source for material taken from the docs

### DISCUSSION THREAD

**Discussion by ghost** [jaredhanson/passport#1024] [Help] [answered]
I need to copy (and translate) some sentences from the Passport documentation found on www.passportjs.org to create some flashcards (with a program called Anki) for personal use. To indicate the source of the sentences is it OK if I write "Adapted from: https://www.passportjs.org/docs/ (Copyright (c) 2011-2021 Jared Hanson)" ?

**Comment by jaredhanson** [comment, marked as answer]
👍  That's fine with me.

**Reply by ghost** [reply]
Ok, thanks!
